### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by DYColdWind**

### CODE REVIEW

File: src/main/resources/assets/matteroverdrive/info/zh_CN/item.contract.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<entry stylesheet="matteroverdrive:info/styles/guide.css">
```
Comment (adrwa) [inline src/main/resources/assets/matteroverdrive/info/zh_CN/item.contract.xml:19]: License header is missing from this file

File: src/main/resources/assets/matteroverdrive/info/zh_CN/item.portable_decomposer.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<entry stylesheet="matteroverdrive:info/styles/guide.css">
```
Comment (adrwa) [inline src/main/resources/assets/matteroverdrive/info/zh_CN/item.portable_decomposer.xml:19]: License header is missing from this file

File: src/main/resources/assets/matteroverdrive/info/zh_CN/tile.contract_market.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<entry stylesheet="matteroverdrive:info/styles/guide.css">
```
Comment (adrwa) [inline src/main/resources/assets/matteroverdrive/info/zh_CN/tile.contract_market.xml:1]: License header is missing from this file

File: src/main/resources/assets/matteroverdrive/lang/zh_cn.lang [hunk 4]
```diff
@@ -1,143 +1,145 @@
...
+item.matteroverdrive.ion_sniper.name=离子狙击枪
+item.matteroverdrive.plasma_shotgun.name=相位散弹枪
+item.matteroverdrive.weapon_receiver.name=弹药接收器
+item.matteroverdrive.weapon_handle.name=武器把手
+item.matteroverdrive.plasma_core.name=相位核心
+item.matteroverdrive.portable_decomposer.name=携带式物质分解仪
+item.matteroverdrive.portable_decomposer.details=将拾取物品分解为等离子物质
+item.matteroverdrive.record.mo.transformation.desc=BoxCat Games - Trace Route
+
+#/#//////////// Upgrades ////////////
```
Comment (adrwa) [inline src/main/resources/assets/matteroverdrive/lang/zh_cn.lang]: Whats with the hashtags in the comments?

**Review by adrwa — CHANGES_REQUESTED** [review]

**Comment by DYColdWind** [comment]
I think，i should complete the revision.
‘src/main/resources/assets/matteroverdrive/info/zh_CN/tile.contract_market.xml’ of ‘License header is missing from this file’ has been folded .
In addition, in 'en_us':item.contract.xml /  item.portable_decomposer.xml also missing ‘License header is missing from this file’ .

**Review by adrwa — APPROVED** [review]

**Pull request merged**